apps/frontend/src/components/form/GrantForm.tsx

```diff
@@ -5,6 +5,27 @@ import React from 'react';
 import FormField from './FormField';
 import { StatesDropdownValues } from '../../enums/DropdownValues';
 
+export const grantFormInitialValues = {
```

Comment:
Are you able to generate this from the schema that we already have defined? Seems like theres a getDefault function on Yup schemas, but I haven't looked too much into it.
We want to limit the number of places we would have to make changes if the schema changes.

Reply:
Done in 7e5aa44.